### PR TITLE
Add cleanup to tests_selinux_disabled.yml test. Tag tests as Tier1.

### PR DESCRIPTION
**Pull request by vcrhonek**

### CODE REVIEW

File: tests/tests_selinux_disabled.yml [hunk 1]
```diff
@@ -81,3 +82,10 @@
         name: sar-user
         remove: yes
         state: absent
```
Comment (pcahyna) [inline tests/tests_selinux_disabled.yml]: I think those three tasks above should be also tagged as `[ 'tests::cleanup', 'tests::tier1::cleanup' ]`. Maybe use a block?

Comment (pcahyna) [reply tests/tests_selinux_disabled.yml]: Applies also to "Backup original /etc/selinux/config", one does not need to back it up, if one does not need to restore it.

Comment (vcrhonek) [reply tests/tests_selinux_disabled.yml]: Fixed.

**Comment by tyll** [comment]
[citest bad]

**Comment by richm** [comment]
@vcrhonek @pcahyna do we still want this PR?  We are carrying a patch downstream that is very similar to this PR.  Why does this use `import_tasks` instead of `include_tasks`?

**Comment by richm** [comment]
closing in favor of https://github.com/linux-system-roles/selinux/pull/77

**Pull request closed without merging**